lib/pushmi_pullyu/cli.rb

```diff
@@ -26,12 +27,19 @@ def parse(argv = ARGV)
   end
 
   def run
-    if options[:daemonize]
-      start_server_as_daemon
-    else
-      # If we're running in the foreground sync the output.
-      $stdout.sync = $stderr.sync = true
-      start_server
+    configure_rollbar
+    begin
+      if options[:daemonize]
+        start_server_as_daemon
+      else
+        # If we're running in the foreground sync the output.
+        $stdout.sync = $stderr.sync = true
+        start_server
+      end
+    # rubocop:disable Lint/RescueException
```

Comment:
You can see them all here, https://ruby-doc.org/core-2.3.0/Exception.html but 2 big classes of errors are the SignalEror (for obvs reasons important with our signal handlers), and ScriptError which syntax errors fall under. Always good to know about Production run time typos